colors/termschool.vim

```diff
@@ -127,3 +127,5 @@ hi SpellRare ctermfg=0 ctermbg=225 guifg=#000000 guibg=#ffdfff
 hi TabLineFill ctermfg=NONE ctermbg=NONE cterm=NONE guifg=NONE guibg=NONE gui=NONE
 hi TabLineSel ctermfg=NONE ctermbg=59 cterm=NONE guifg=NONE guibg=#3f4b52 gui=NONE
 hi TabLine ctermfg=NONE ctermbg=233 cterm=NONE guifg=NONE guibg=#121212 gui=NONE
+hi SignColumn     ctermbg=NONE guibg=#2e373b
```

Comment:
Yes, you're correct when you say that anything between #000000 and #ffffff is valid for gvim, however, the idea of this theme is to make the terminal experience as close as possible to the gui experience. There are a *ton* of good gvim themes, but they all behave ghastly when used with the terminal. :) That's why I think we should use the corresponding color.